python/sauron-core/core/analysis/classifier.py

```
@@ -0,0 +1,24 @@
+import numpy as np
+from sklearn.svm import SVC
+from sklearn.preprocessing import StandardScaler
+from typing import Dict
+
+class AIImageClassifier:
+    def __init__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)